content/participants/ttjc@Sarthakux/index.md

```diff
@@ -34,7 +34,61 @@ It would be good if someone helps me with implementing an authentication in my a
 - [LinkedIn](https://www.linkedin.com/in/sarthak-yadav-95a114184/)
 
 ### projects
-I have not done any projects presently but will surely update very soon with real links when done.
+My projects:
+
+### Quiz APP
+
+ _description_ A simple quiz app made by HTML,CSS AND JS.
+ _hosted link_ https://sy-quiz.netlify.app
+
+_github link_ https://github.com/Sarthakux/quiz1
```

Comment:
put proper spacing and change a line after each so that markdown is rendered properly view the file for more details. https://github.com/tanaypratap/teamtanay.jobchallenge.dev/blob/5bff09febd4c81380dafcda9e9040df036d72cb5/content/participants/ttjc%40Sarthakux/index.md

Reply:
do the same throughout the list.